.github/ISSUE_TEMPLATE/01-sdk-bug.yml

```diff
@@ -0,0 +1,100 @@
+name: SDK Bug Report
+description: File SDK bug report
+labels: ["type:bug", "status:waiting-for-triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Thank you for taking the time to fill out this bug report!**
+        💥Before submitting a new request, please search existing issues to see if an issue already exists.
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: |
+        Provide a description of the actual behavior observed. If applicable please include any error messages, exception stacktraces or a screenshot.
+      placeholder: I am trying to do [...] but [...]
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: |
+        A clear and concise description of what you expected to happen.
+      placeholder: Expected behavior
+    validations:
+      required: true
+  - type: textarea
+    id: repro-steps
+    attributes:
+      label: How to reproduce
+      description: |
+        Please include minimal steps to reproduce the problem if possible. E.g.: the smallest possible code snippet; or steps to run project in link above. If possible include text as text rather than screenshots (so it shows up in searches).
+        If there's a link to a public repo where the sample code exists, include it too.
+      placeholder: Minimal Reproduction steps
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: SDK Version
+      placeholder: e.g. 5.32.1
+      description: Version of the SDK with the bug described above.
+    validations:
+      required: false
```

Comment:
```suggestion
      required: true
```

Reply:
We agreed as a team to have this as optional incase there is no previously known version known to work.